rules/aip0142/duration_offset_comment.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aip0142
+
+import (
+	"strings"
+
+	"github.com/googleapis/api-linter/lint"
+	"github.com/googleapis/api-linter/rules/internal/utils"
+	"github.com/jhump/protoreflect/desc"
+)
+
+var durationOffsetComment = &lint.FieldRule{
+	Name: lint.NewRuleName(142, "duration-offset-comment"),
+	OnlyIf: func(f *desc.FieldDescriptor) bool {
+		return utils.GetTypeName(f) == "google.protobuf.Duration" && strings.HasSuffix(f.GetName(), "_offset")
+	},
+	LintField: func(f *desc.FieldDescriptor) []lint.Problem {
+		comment := f.GetSourceInfo().GetLeadingComments()
+		if comment == "" {
```

Comment:
We already have https://linter.aip.dev/192/has-comments so I'm not sure if this is the most useful heuristic. If there is a comment `// Start offset` this rule does not help us with the guidance it is intended to enforce.

Is there another heuristic that we can use to evaluate a comment for its compliance? Do you think `strings.Contains(comment, "relative")` would be useful/accurate enough? Perhaps take a look at existing fields in googleapis.